superset-frontend/plugins/legacy-plugin-chart-world-map/src/controlPanel.ts

```diff
@@ -106,7 +107,25 @@ const config: ControlPanelConfig = {
           },
         ],
         ['color_picker'],
+        [
+          {
+            name: 'color_by',
+            config: {
+              type: 'RadioButtonControl',
+              label: t('Color by'),
+              default: ColorBy.metric,
+              options: [
+                [ColorBy.metric, t('metric')],
+                [ColorBy.country, t('country')],
```

Comment:
```suggestion
                [ColorBy.metric, t('Metric')],
                [ColorBy.country, t('Country')],
```
Nitpicking. This really doesn't matter since the CSS capitalizes it. But if we ever move away from all-caps in the design, then this will be nicely capitalized that day ;)